kafka_consumer_freshness_tracker/README.md

```diff
@@ -47,7 +47,12 @@ bazel-bin/kafka_consumer_freshness_tracker/src/main/java/com/tesla/data/consumer
 
 Bundled as a jar you can run the Tracker with a configuration file with the `--conf <path to conf>` flag. For local 
 testing, you can also just add the `--once` flag to just run the Tracker once, and then dump the metrics to the console, 
-before stopping.
+before stopping. 
+The `--strict` flag can be used to run the Tracker in strict mode, which enforces stricter validation of the cluster
+configurations. With `--strict` enabled, the application will crash if the bootstrap servers specified for the Tracker do
```

Comment:
It will also crash if burrow is unreachable during setup, based on the code below.

Reply:
good point

Reply:
fixed